docs/source/markdown/podman-run.1.md.in

```diff
@@ -477,7 +477,7 @@ tmpfs directories on _/run_ and _/tmp_.
 ```
 $ podman run --read-only -i -t fedora /bin/bash
 
-$ podman run --read-only --read-only-tmpfs=false --tmpfs /run -i -t fedora /bin/bash
+$ podman run --read-only --read-write-tmpfs=false --tmpfs /run -i -t fedora /bin/bash
```

Comment:
(commenting on code that github does not show by default, so please click the uparrow-on-bar to see lines 473-475). Those lines read:
>The best way to handle this is to mount tmpfs directories on _/run_ and _/tmp_.

This looks like a holdover from a previous age: the implication is that `/run` and `/tmp` are also read-only. I haven't taken the time to go through history, but is it possible that the `read-only-tmpfs` option (the one that is being renamed here) was added after this was written, and that this used to be true but no longer is? Can you re-review lines 473-475 and see if they should be rewritten, maybe something like

>Read-only containers still mount `/run`, `/tmp`, `/var/tmp`, `/dev`, and `/dev/shm` as read-write tmpfs. Use `--read-write-tmpfs=false` to protect even `tmpfs` directories, and `--tmpfs=PATH` to explicitly limit the read-write tmpfs filesystems.

(Idea only. That reads horribly. My documentation-writing brain is not fully engaged yet).